docs/eventing/sources/sinkbinding/getting-started.md

```diff
@@ -0,0 +1,300 @@
+---
+title: "Create a SinkBinding object"
+weight: 02
+type: "docs"
+---
+
+![API version v1](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
+
+This topic describes how to create a SinkBinding object and connect it to a
+subject in your cluster.
+
+If you follow the examples in this topic, you will have a SinkBinding object
+that resolves a Knative Service (the sink) into a URI, and sets that URI in the
+environment variable `K_SINK` in new jobs from a cron job (the subject).
+The SinkBinding updates `K_SINK` if the URI changes.
```

Comment:
reword to remove future tense and (parentheses)?

e.g.
The following example helps you to create a SinkBinding object that resolves a Knative service as a URI, and sets the URI in the environment variable `K_SINK`. The SinkBinding updates `K_SINK` if the URI is changed.

> in new jobs from a cron job (the subject).

This bit wasn't really clear to me what it means?